generate-stackbrew-library.sh

```diff
@@ -83,7 +83,8 @@ for branch in "${branches[@]}"; do
 					(
 						if contains("alpine") then 1
 						elif contains("corretto") then 2
-						elif contains("graal") then 3
+						elif contains("ubi") then 3
```

Comment:
maybe you could differentiate between ubi images? I think currently there are ubi8, ubi9 and ubi10 in beta

Reply:
This script is just for generating the file I need for the official images repo. For now, I only planned on adding 9, then I'd add 10 when it's out. Will that work?

Reply:
yes it will... I mean I mentioned it only in this place...but if you know that there will be ubi9 and ubi10 just take a look in other places and if that makes sense rename it